pkg/virt-api/api.go

```diff
@@ -882,13 +882,6 @@ func (app *virtAPIApp) Run() {
 	// Run informers for webhooks usage
 	kubeInformerFactory := controller.NewKubeInformerFactory(app.virtCli.RestClient(), app.virtCli, app.aggregatorClient, app.namespace)
 
-	configMapInformer := kubeInformerFactory.ConfigMap()
```

Comment:
This needs to be moved somewhere else otherwise we have regression.

Reply:
Thanks for the review @xpivarc. Not sure I get this point. Why would we have regression? The CM is no longer in use.

Reply:
Checked again. This informer is only watches the obsolete configmap. I don't understand how it will cause regression.

Reply:
@nunnatsa  Sorry for my not exact comment. I meant to comment on the line below. The important part is `apiHealthVersion.Clear()` in the handler. We need to move it to another handler. Meaning we need a handler on another informer.

It is an optimization so we would not make a frequent API call to figure out if `kube-api` is available. This is most important in `virt-handler` as the count of them can be very high.

Reply:
@xpivarc - thanks. got it - fixed.